src/com/company/Contact.java

```diff
@@ -0,0 +1,23 @@
+package com.company;
+
+public class Contact {
+    String name;
+    String telephone;
+    String email;
+
+    Contact(String nameContact, String telephoneContact, String emailContact){
+        name=nameContact;
+        telephone=telephoneContact;
+        email=emailContact;
+    }
+
+    public String giveName(){
```

Comment:
@SFDC-Melby-Arguedas Correct name mehotd would be **getName** because it is a getter for name